### PR TITLE
Bug bounty page

### PR DESCRIPTION
**Pull request by StanislavBreadless**

### CODE REVIEW

File: docs/dev/bug-bounty.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- You must be able to verify a signature from the same address
+- Provide enough information about the vulnerability
+
+## Bounty Payout
+
+![bounty-table.png](https://zksync.io/bounty-table.png)
+
+## Contact
+
+Please submit all your requests here: [security@zksync](mailto:security@zksync).
```
Comment (popzxc) [inline docs/dev/bug-bounty.md]: Shouldn't it be `zksync.io`?

File: docs/faq/security.md [hunk 2]
```diff
@@ -125,3 +125,7 @@ trust assumptions altogether by embracing a transparent zero-knowledge proof sys
 ## Security audits
 
 [Security audits](/updates/security-audits.md) are being conducted before each major update.
+
+## Bug Bounty Program
+
+Besides security audits, we offer bug bounty program. You can read more about it [here](/dev/bug-bounty).
```
Comment (popzxc) [inline docs/faq/security.md:131]:
```suggestion
Besides security audits, we have a Bug Bounty program. You can read about it [here](/dev/bug-bounty).
```

**Review by popzxc — APPROVED** [review]

File: docs/faq/security.md (hunk 2 above)
Comment (StanislavBreadless) [reply docs/faq/security.md:131]: In private conversation it was decided that it is hard to determine which is the right way to do it, so I'll keep it as it is

**Pull request merged**